dbfit-java/core/src/main/java/dbfit/fixture/StatementExecution.java

```diff
@@ -19,7 +19,11 @@ public void registerOutParameter(int index, int sqlType, boolean isReturnValue)
 
     public void setObject(int index, Object value, int sqlType, String userDefinedTypeName) throws SQLException {
         if (value == null) {
-            statement.setNull(index, sqlType, userDefinedTypeName);
+            try {
+                statement.setNull(index, sqlType, userDefinedTypeName);
+            } catch (UnsupportedOperationException e) { // Former procedure is not supported in Sybase
```

Comment:
I have some odd gut feeling about using exceptions handling for some logic. I'm wondering can we check if`userDefinedTypeName` is null instead?

Reply:
Hi,

1. Sure, I'll try checking userDefinedTypeName for null instead of using try catch block.
2. Regarding testing on Travis-CI. I'd like to set it up, although I need a bit of guidance here. Could you give me some links/hints on how to upload a pre-built (with sybase pre-installed) docker image to Travis-CI? Just some notes, I'll prepare the image myself. By the way, is there a way to access Oracle or DB2 images used in Travis?

Thanks for help in advance. I hope I'll be able to improve this Pull request to a mergeable state :)

Reply:
We're using docker-based environments for Oracle and DB2 but the images themselves are not published on Travis CI. We're just pulling the images from [dockerhub](https://hub.docker.com/) - you may find them there. E.g.: https://hub.docker.com/r/wnameless/oracle-xe-11g/

So if you manage to publish your docker image to dockerhub - it can be accessed in a similar way.

Reply:
hello, can we connect to Sybase now with latest dbFit versoin?

Reply:
@martynas There's a Sybase Docker image here. https://hub.docker.com/r/nguoianphu/docker-sybase/ I've just started using it to test this PR from my end.